app/src/ui/app.tsx

```diff
@@ -535,7 +535,19 @@ export class App extends React.Component<IAppProps, IAppState> {
   }
 
   public componentDidMount() {
-    document.ondragover = document.ondrop = e => e.preventDefault()
+    document.ondragover = e => {
+      if (this.state.currentPopup != null) {
```

Comment:
Could we use `this.isShowingModal` here instead so that we handle the case where an error dialog is showing? And when you make that change I'd really appreciate it if you could clean up our old sins and apply the same fix to the condition inside of the `document.body.ondrop` handler as well.

Reply:
@niik Some questions so I can clarify some things before I dive into the changes you suggested.

> Could we use this.isShowingModal here instead so that we handle the case where an error dialog is showing?

So you say to replace `this.state.currentPopup` with `this.isShowingModal`? or add it to the if statement. I think adding it would be a better "solution" for the reason that we will do more checks, one for the popup & the other for the error dialog but not sure 🤔 

> And when you make that change I'd really appreciate it if you could clean up our old sins

What you mean by that?

Reply:
**Update**:  I think I understood what you meant. Just updated the PR!